src/client/components/pages/collection.js

```diff
@@ -336,7 +349,8 @@ CollectionPage.propTypes = {
 	isCollaborator: PropTypes.bool,
 	isOwner: PropTypes.bool,
 	nextEnabled: PropTypes.bool.isRequired,
-	size: PropTypes.number
+	size: PropTypes.number,
+	userId: PropTypes.number.isRequired
```

Comment:
I'm not sure this should be required. What if I'm not logged in? I should still be able to see public collections.

In src/server/routes/collection.js, you have `const userId = parseInt(req.user?.id, 10);` which will return `NaN` if user is undefined. Perhaps it should be added to the props conditionally ?